system/k3s/k3s.SlackBuild

```diff
@@ -72,7 +87,7 @@ mkdir -p usr/bin
 install -m 0755 $CWD/$PRGNAM usr/bin/$PRGNAM
 
 mkdir -p etc/rc.d
-install -m 0644 $CWD/config/rc.k3s etc/rc.d/rc.k3s
```

Comment:
This should remain 0644 and the admin can adjust as required. It should also be installed with `.new` and managed in your doinst.sh

Reply:
The if an admin sets it to 0755, the permissions will be preserved during upgrades.

Reply:
Ah, yes, this must have changed during my testings
Gonna fix it